src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -92,12 +101,48 @@ private void validateTrustDetails(OverseasEntitySubmissionDto overseasEntitySubm
     }
 
     public Errors validatePartial(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+        if (isRoeUpdateEnabled && overseasEntitySubmissionDto.isForUpdate()) {
+            validatePartialUpdateDetails(overseasEntitySubmissionDto, errors, loggingContext);
+        } else {
+            validatePartialRegistrationDetails(overseasEntitySubmissionDto, errors, loggingContext);
+        }
+        return errors;
+    }
+
+    public Errors validatePartialUpdateDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+
+        var entityDto = overseasEntitySubmissionDto.getEntity();
+
+        if (Objects.nonNull(entityDto)) {
+            var entityEmail = entityDto.getEmail();
```

Comment:
So we are allowing empty emails for Update? Why?

Reply:
Think I would rather skip the entity validation if we really want to allow this. I think it is a bug in the UI that you can submit without an email address for the OE being entered. Not sure we want to mask that here. But needs a comment as TODO at least?

Reply:
So this has to do with the email being private data and currently is not getting fetched at the time the OE is fetched from the public data.... could be removed after both public and private data fetches are implemented

Reply:
So we do need to disable this for now and let no email through. Should maybe add a comment to say that this is temporary.

Reply:
comment added to code.